sphinx/tutorial/cylc/runtime/introduction.rst

```diff
@@ -0,0 +1,390 @@
+
+It is often a good idea to keep our scripts with the cylc suite rather than
+leaving them somewhere else on the system. If you create a ``bin`` directory
+within the :term:`suite directory` this directory will be added to the path
+when tasks run. E.G:
+
+.. code-block:: bash
+   :caption: bin/hello-world
+
+   #!/usr/bin/bash
+   echo 'Hello World!'
+
+.. code-block:: cylc
+   :caption: suite.rc
+   :emphasize-lines: 3
+
+   [runtime]
+       [[hello_world]]
+           script = hello_world
+
+
+Running A Suite
+---------------
+
+It is a good idea to check the suite for errors before running it.
+Cylc provides a command which automatically checks for any obvious
+configuration issues called ``cylc validate``.
+
+.. code-block:: sub
+
+   cylc validate <path/to/suite>
+
+Next we run the suite using the ``cylc run`` command.
+
+.. code-block:: bash
+
+   cylc run <name>
+
+The ``name`` is the name of the :term:`suite directory` (i.e. if we create a
+suite in ``~/cylc-run/foo`` then the ``<name>`` would be ``foo``).
+
+.. note::
+
+   In this tutorial we are writing our suites in the ``cylc-run`` directory.
+
+   It is possible to write them elsewhere on the system. If we do so we
+   must register the suite with cylc before use.
+
+   We do this using the ``cylc reg`` command which we supply with a name which
+   will be used to refer to the suite in place of the path i.e:
+
+   .. code-block:: sub
+
+      cylc reg <name> <path/to/suite>
+      cylc validate <name>
+      cylc run <name>
+
+   The ``cylc reg`` command will create a directory for the suite in the
+   ``cylc-run`` directory meaning that we will have separate
+   :term:`suite directories <suite directory>` and
+   :term:`run directories <run directory>`.
+
+.. _tutorial-tasks-and-jobs:
+
+Tasks And Jobs
+--------------
+
+When a :term:`task` is "Run" it creates a :term:`job`. The job is a bash
+file containing the script you have told the task to run along with
+configuration and a system for trapping errors. It is this :term:`job`
+which actually gets executed. This "job file" is called the :term:`job script`.
+
+During its life a typical :term:`task` goes through the following states:
+
+Waiting
+   :term:`Tasks <task>` wait for their dependencies to be satisfied before
+   running, in the mean time they are in the "Waiting" state.
+Submitted
+   When the :term:`task's <task>` dependencies have been met it is ready for
+   submission. During this phase the :term:`job script` is created.
+   The :term:`job` is then submitted to the specified batch system, more on
+   this in the :ref:`next section <tutorial-batch-system>`.
+Running
+   A :term:`task` is in the "Running" state as soon as the :term:`job` is
+   executed.
+Succeeded
+   If the :term:`job` the :term:`task` submitted has successfully completed
+   (zero return code) then it is said to have succeeded.
+
+These are called the :term:`task states <task state>` and there are a few more
+(e.g. failed).
```

Comment:
I decided only to cover the main states in the success scenario for simplicity. I didn't want to write out a comprehensive list as it would be distracting from the content / hard to understand at this stage. The omitted task states are:

- held
- ready
- queued
- failed
- runahead
- retrying
- submit-retrying
- submit-failed
- expired